Makefile

```diff
@@ -0,0 +1,50 @@
+.PHONY: all clean generate fmt
+.PHONY: tidy get build test up
+
+GO ?= go
+GOFMT ?= gofmt
+GOFMT_FLAGS = -w -l -s
+GOGENERATE_FLAGS = -v
+
+GOBIN ?= $(GOPATH)/bin
+
+REVIVE ?= $(GOBIN)/revive
+REVIVE_FLAGS ?= -formatter friendly
+REVIVE_INSTALL_URL ?= github.com/mgechev/revive
+
+TMPDIR ?= .tmp
+
+all: get generate fmt tidy build
```

Comment:
Just wondering if fmt should not be after tidy. On the other hand tidy is not touching .go files only .mod and .sum as opposed to fmt which just touches .go files. So probably it is fine.

Reply:
`tidy` already calls `fmt` so I've removed the explicit reference in `all`